api/src/services/sheets/sheets.js

```diff
@@ -1,54 +1,19 @@
-import { google } from 'googleapis'
-import fs from 'fs'
-
 import { requireAuth } from 'src/lib/auth'
 import {
   updateRestaurant,
   restaurantByOwnerId,
+  restaurantById,
 } from '../restaurants/restaurants'
 
+import {
+  getBusInfoFromGoogleAPI,
+  createSheetGoogleAPI,
+} from '../googleapi/googleapi'
+
 export const createSheet = async () => {
```

Comment:
Let's move this to the restaurant service. We could call it `createInitialSheet` or something. If once we move this and `readSheet`, we can remove the sheets service entirely!